#!/bin/sh /usr/share/dpatch/dpatch-run
## 08-manpage-drop.dpatch by Daniel Baumann <daniel@debian.org>
##
## DP: Don't use confusing word 'drop' when 'delete' is meant (Closes: #134100).

@DPATCH@

diff -Naurp dosfstools.orig/dosfsck/dosfsck.8 dosfstools/dosfsck/dosfsck.8
--- dosfstools.orig/dosfsck/dosfsck.8	2008-06-27 06:12:18.000000000 +0200
+++ dosfstools/dosfsck/dosfsck.8	2008-06-27 06:27:03.000000000 +0200
@@ -24,15 +24,15 @@ Bad clusters (read errors). The clusters
 removed from files owning them. This check is optional.
 .IP \-
 Directories with a large number of bad entries (probably corrupt). The
-directory can be dropped.
+directory can be deleted.
 .IP \-
-Files . and .. are non-directories. They can be dropped or renamed.
+Files . and .. are non-directories. They can be deleted or renamed.
 .IP \-
-Directories . and .. in root directory. They are dropped.
+Directories . and .. in root directory. They are deleted.
 .IP \-
 Bad file names. They can be renamed.
 .IP \-
-Duplicate directory entries. They can be dropped or renamed.
+Duplicate directory entries. They can be deleted or renamed.
 .IP \-
 Directories with non-zero size field. Size is set to zero.
 .IP \-
@@ -88,8 +88,8 @@ MS-DOS uses only 0xfff7 for bad clusters
 0xfff0...0xfff7 are for this purpose (but the standard value is still
 0xfff7).
 .IP \fB\-d\fP
-Drop the specified file. If more that one file with that name exists, the
-first one is dropped.
+Delete the specified file. If more that one file with that name exists, the
+first one is deleted.
 .IP \fB\-f\fP
 Salvage unused cluster chains to files. By default, unused clusters are
 added to the free disk space except in auto mode (\fB-a\fP).
